core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1771,11 +1773,13 @@ public long getNewBlockIdForFile(AlluxioURI path) throws FileDoesNotExistExcepti
          LockedInodePath inodePath = mInodeTree.lockFullInodePath(path, LockPattern.WRITE_INODE);
          FileSystemMasterAuditContext auditContext =
             createAuditContext("getNewBlockIdForFile", path, null, inodePath.getInodeOrNull())) {
-      try {
-        mPermissionChecker.checkPermission(Mode.Bits.WRITE, inodePath);
-      } catch (AccessControlException e) {
-        auditContext.setAllowed(false);
-        throw e;
+      if (!skipFilePermissionCheck(inodePath)) {
+        try {
+          mPermissionChecker.checkPermission(Mode.Bits.WRITE, inodePath);
```

Comment:
This permission check could do other checks, it is not safe to skip here.

Reply:
a possible solution is to escalate the permission temporarily, then revoke the permission after this check is done.

Reply:
eg: escalate to 600 before the check if it is a newly created, then degrade to 400.

Reply:
> This permission check could do other checks, it is not safe to skip here.

Yeah, skipping all checks is not a good approach.

For escalation, I searched keywords "escalate" and "escalation" in the source code, and it seems that no places using this mechanism currently.

To implement escalation, I had two ideas
1. Add "write" to the file's permission and change it back after checking. This is quite dirty and dangerous.
2. Add a new parameter for related methods to store the escalated permission. I checked and found it needs to change a lot of methods in several classes.

So I choose to decrease the permission to be checked.